apps/ui/src/common/pages/birth.tsx

```diff
@@ -219,7 +220,13 @@ const Page: FC<PageProps> = ({ location, signInHRef }) => {
                 !results && !details ? (
                   <div style={{ textAlign: 'center' }}>
                     <h4>Did you know?</h4>
-                    <p>Did you know you can enter short dates?</p>
+                    <p>Dates can be entered as:</p>
+                    <ul className="govuk-list">
+                      <li>15072018</li>
+                      <li>15/07/2018</li>
+                      <li>15.07.2018</li>
+                      <li>2018-07-15</li>
+                    </ul>
                   </div>
```

Comment:
Not sure how it conforms with the design pattern but it seems to me that this should be a valid way of entering a date:

![date](https://user-images.githubusercontent.com/22613578/132673167-8d2c4064-fb23-4cc1-b1a8-6f938d357561.png)

Reply:
I think this date information should be separated into a different component at some point too as it'll be used in other datasets

Reply:
Have added a rule for dates formatted as dd-mm-yyyy so these are now accepted.